framework/Source/Operations/HistogramEqualization.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HistogramEqualization.swift
+//
+//
+//  Created by Chiaote Ni on 2024/6/4.
+//
+
+import Foundation
+
+public class HistogramEqualization: ImageProcessingOperation {
+    public var maximumInputs: UInt = 1
+    public let targets = TargetContainer()
+    public let sources = SourceContainer()
```

Comment:
do we need to public these?

Reply:
Yes, we need them. 
These 2 properties are defined in the protocol ImageProcessingOperation, and the HistogramEqualization isn't inherited from the BasicOperation because it won't use any metal shader.